### ISSUE TITLE
repositoryPathPattern breaks basic-code-intel repo-scoped symbol search queries in the browser extension

### ISSUE THREAD

**Issue by mrnugget** [sourcegraph/sourcegraph#4414]
I'm reviewing https://github.com/sourcegraph/sourcegraph/pull/4403/files and when looking at `enterprise/cmd/frontend/internal/authz/bitbucketserver/provider_test.go` I wanted to hover over `fakeBitbucketServerAPI` to see its definition.

But no tooltip showed up. Then hover didn't show up for other types as well. A reload fixed that last part though. But still: hovering over `fakeBitbucketServerAPI` resulted in no tooltip.

Live footage:

![extension_no_hover](https://user-images.githubusercontent.com/1185253/59092517-05215d00-8912-11e9-839a-6d1c287b1174.gif)

**Comment by lguychard** [comment]
I tried to reproduce this, but I actually got super quick hovers on all symbols on that PR, including `fakeBitbucketServerAPI`.

![image](https://user-images.githubusercontent.com/1741180/59093213-a78e1000-8913-11e9-9003-d7a7c279774e.png)

Then I tried to point my browser extension to sourcegraph.sgdev.org, and was no longer getting hovers for `fakeBitbucketServerAPI`.

@mrnugget which Sourcegraph instance is your browser extension pointing to?

@chrismwendt any idea of what might be causing this discrepancy?

**Comment by mrnugget** [comment]
> @mrnugget which Sourcegraph instance is your browser extension pointing to?

`sourcegraph.sgdev.org`

**Comment by lguychard** [comment]
When visiting [`sourcegraph.sgdev.org/sourcegraph/sourcegraph@core/bitbucket-server-authz`](
https://sourcegraph.sgdev.org/sourcegraph/sourcegraph@core/bitbucket-server-authz/-/blob/enterprise/cmd/frontend/internal/authz/bitbucketserver/provider_test.go#L275:11), I get hovers on `fakeBitbucketServerAPI`:

![image](https://user-images.githubusercontent.com/1741180/59094959-a2cb5b00-8917-11e9-8f37-47f9b24a34d2.png)

On the browser extension side, all looks good, but no symbol search results are returned for `fakeBitbucketServerAPI` on sgdev: 

```
{"data":{"search":{"results":{"__typename":"SearchResults","limitHit":false,"results":[]}}}}
```

Query on `sourcegraph.sgdev.org`: `^fakeBitbucketServerAPI$ case:yes file:.(go)$ type:symbol repo:^sourcegraph/sourcegraph$@be2b5d83abef843cb254840270e9c23711bc65da`

Query in the browser extension: `^fakeBitbucketServerAPI$ case:yes file:.(go)$ type:symbol repo:^github.com/sourcegraph/sourcegraph$@be2b5d83abef843cb254840270e9c23711bc65da`

Notice the repo name is different between the two queries: in the webapp, the repo name as affected by `repositoryPathPattern` is used. In the browser extension, the "inferred" repo name is used. Everything works well on sourcegraph.com, where no `repositoryPathPattern is set`. For symbols that get hovers, the search query is not scoped to the repo: `^Metadata$ case:yes file:.(go)$ type:symbol`.

tl;dr: this is another one that falls under https://github.com/sourcegraph/sourcegraph/issues/4370
